<launch>
    <node pkg="eac_pkg" type="stm32_node" name="stm32_node"/>
    <node pkg="eac_pkg" type="object_inference.py" name="object_inference" />
    <node pkg="eac_pkg" type="zone_filter.py" name="zone_filter"/>
    <node pkg="eac_pkg" type="main_node" name="main_node"/>
    <node pkg="eac_pkg" type="schema2_node" name="schema2_node"/>
</launch>
